port/win/env_win.cc

```diff
@@ -872,6 +872,10 @@ void  WinEnvThreads::SleepForMicroseconds(int micros) {
   std::this_thread::sleep_for(std::chrono::microseconds(micros));
 }
 
+void WinEnvThreads::SleepForNanoseconds(int nanos) {
+  std::this_thread::sleep_for(std::chrono::nanoseconds(nanos));
```

Comment:
@yuslepukhin We are not trying to achieve a nano-level granularity here. A sleep system call would never achieve nano-level granularity on any platforms. So here it does NOT make any difference for us.